hs_tools_resource/templates/pages/toolresource.html

```diff
@@ -19,6 +19,76 @@ <h4><strong>Home Page URL:</strong></h4>
                         <hr style="border:0">
                     {%  endif %}
                 </div>
+                <div>
+                    {% if testing_protocol_url %}
+                        <h4><strong>Testing Protocol URL:</strong></h4>
+                        <hr style="margin-top: 0px;margin-bottom: 2px">
+                        <div class="row">
+                            <div class="col-sm-8">{{ testing_protocol_url.value|urlize }}</div>
+                        </div>
+                        <hr style="border:0">
+                    {%  endif %}
+                </div>
+                <div>
+                    {% if help_page_url %}
+                        <h4><strong>Help Page URL:</strong></h4>
+                        <hr style="margin-top: 0px;margin-bottom: 2px">
+                        <div class="row">
+                            <div class="col-sm-8">{{ help_page_url.value|urlize }}</div>
+                        </div>
+                        <hr style="border:0">
+                    {%  endif %}
+                </div>
+                <div>
+                    {% if source_code_url %}
+                        <h4><strong>Source Code URL:</strong></h4>
+                        <hr style="margin-top: 0px;margin-bottom: 2px">
```

Comment:
@aphelionz It's probably better to put this style definition above in css since it is used in many places in this template.